libraries/GCS_MAVLink/GCS.h

```diff
@@ -74,7 +74,7 @@ enum ap_message {
     MSG_BATTERY_STATUS,
     MSG_AOA_SSA,
     MSG_LANDING,
-    MSG_RETRY_DEFERRED // this must be last
+    MSG_RETRY_DEFERRED
```

Comment:
Ah, but with these patches that array doesn't *need* to be that size any more :-)